UI/src/controller/massTransitStopsCollection.js

```diff
@@ -28,14 +29,35 @@
     };
 
     var refreshAssets = function(mapMoveEvent) {
-      backend.getAssetsWithCallback(mapMoveEvent.bbox, function(backendAssets) {
+      var treatAssets = function (backendAssets) {
         backendAssets = filterComplementaries(backendAssets);
         if (mapMoveEvent.hasZoomLevelChanged) {
           eventbus.trigger('assets:all-updated massTransitStops:available', backendAssets);
         } else {
           eventbus.trigger('assets:new-fetched massTransitStops:available', filterNonExistingAssets(backendAssets, assets));
         }
-      });
+      };
+
+      if(showServicePoints) {
+        backend.getAssetsWithCallbackServiceStops(mapMoveEvent.bbox, treatAssets);
+      }
+      backend.getAssetsWithCallback(mapMoveEvent.bbox, treatAssets);
+      verificationCollection.fetch(mapMoveEvent.bbox, mapMoveEvent.center, massTransitStopTypeId, true);
+    };
+
+    var refreshAssetsServiceStops = function(mapMoveEvent) {
+      var treatAssets = function (backendAssets) {
+        backendAssets = filterComplementaries(backendAssets);
```

Comment:
Are these service points located on links or just an (x, y)? If they are not on links this can be removed

Reply:
These service points are located on a (x, y) so this line will be removed